src/style.css

```diff
@@ -1,3 +1,80 @@
+html {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
 body {
   margin: 0;
```

Comment:
CSS: Don't use tag names for styling (except html and body). Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`